.gitignore

```diff
@@ -26,3 +26,12 @@ pom.xml
 /.lein-failures
 
 config.edn
+
+# MongoDB files
+*.wt
+*.lock
+*.turtle
+data/WiredTiger
+data/diagnostic.data
+data/journal
+data/storage.bson
```

Comment:
What are all these files?

Reply:
For me, 'lein fetch' created the following:
```
        data/WiredTiger
	data/WiredTiger.lock
	data/WiredTiger.turtle
	data/WiredTiger.wt
	data/WiredTigerLAS.wt
	data/_mdb_catalog.wt
	data/collection-0-5099375551159741750.wt
	data/collection-10-5099375551159741750.wt
	data/collection-12-5099375551159741750.wt
	data/collection-14-5099375551159741750.wt
	data/collection-2-5099375551159741750.wt
	data/collection-4-5099375551159741750.wt
	data/collection-6-5099375551159741750.wt
	data/collection-8-5099375551159741750.wt
	data/diagnostic.data/
	data/index-1-5099375551159741750.wt
	data/index-11-5099375551159741750.wt
	data/index-13-5099375551159741750.wt
	data/index-15-5099375551159741750.wt
	data/index-3-5099375551159741750.wt
	data/index-5-5099375551159741750.wt
	data/index-7-5099375551159741750.wt
	data/index-9-5099375551159741750.wt
	data/journal/
	data/mongod.lock
	data/sizeStorer.wt
	data/storage.bson
```
Looks like WiredTiger one of a number of storage engines for mongo:
https://docs.mongodb.com/manual/core/storage-engines/
and it's the default since 3.2

I'm not sure why they wouldn't be showing up for other people.